quinn-h3/src/qpack/prefix_string/decode.rs

```diff
@@ -78,6 +75,23 @@ impl HuffmanDecoder {
     }
 }
 
+fn extract_bits(src: &[u8], mut byte_offset: u32, mut bit_offset: u32, len: u32) -> Result<u8, ()> {
+    if len == 0 || len > 8 || src.len() as u32 * 8 < byte_offset * 8 + bit_offset + len {
+        return Err(());
+    }
+
+    byte_offset += bit_offset / 8;
+    bit_offset -= (bit_offset / 8) * 8;
+    Ok(if bit_offset + len <= 8 {
+        (u8::from_be(src[byte_offset as usize]) << bit_offset) >> (8 - len)
```

Comment:
`u8::from_be` is the identity function; endianness only exists between bytes

Reply:
Yeah, forgot to cut that out.